### ISSUE TITLE
strange duplication bug

### ISSUE THREAD

**Issue by zed-hex** [Azanor/thaumcraft#1881]
Strange duplication bug when a retriever from “thermal dynamics” or a retrieval node from “extra utilities” (the node it needs the stack up grade) pulls from a hungry chest from “thaumcraft” I can’t really explain it. Here is a print screen of the quick set up I made when testing the bug http://prntscr.com/8kille .

**Comment by Azanor** [comment]
The hungry chest is just a copy of the vanilla chest, so this is something you might want to report to those mods.

**Comment by bosto2** [comment]
That is odd considering that this works with no other inventory (including vanilla chests)

**Comment by valveo** [comment]
I tested this with iron chests (all of them), vacuum chest, vanilla chest. No issues. Hungry chest -> duplication. 
Put one item to hungry chest, placed normal chest 3 block away, connected with thermal dynamics duct and then installed retriever to destination point. In few moments chest was full of item I placed to hungry chest.
However item in hungry chest bugged, i couldnt place it or anything. Stack size was 65 when it should be max 64.
